src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.cs

```diff
@@ -40,6 +40,8 @@ private IncrementalHash(HashAlgorithmName name, HMACCommon hmac)
         /// </summary>
         public HashAlgorithmName AlgorithmName => _algorithmName;
 
```

Comment:
Please add triple slash documentation on top of this and all other new public APIs in this PR.